trajectory_executor/com_serial.py

```diff
@@ -175,15 +174,30 @@ def sensor_data_request_callback(self, request: SensorDataRequest, response: Sen
             response.at_goal = data_packet['at_goal']
             break
           else:
-            # bad packet, retry request
-            return self.sensor_data_request_callback(request, response)
+            # bad packet, set values to -1 and let executor handle it
+            # TODO: We should probably add a "error" value in the message for
+            #       the executor to use
+            response.roll = -1
+            response.pitch = -1
+            response.yaw = -1
+            response.l_hip = -1
+            response.l_knee = -1
+            response.r_hip = -1
+            response.r_knee = -1
+            response.l_shoulder = -1
+            response.l_elbow = -1
+            response.r_shoulder = -1
+            response.r_elbow = -1
+            response.at_goal = -1
```

Comment:
What happens if the response position is at -1?

Reply:
I'm thinking that the executor node will check is if any of the values are -1 and if they are then it knows there was an error and sends another request or something.

Reply:
Ahh ok so they're just valued in `[0, 360]` degrees, makes sense!